modules/va_notify/app/controllers/va_notify/callbacks_controller.rb

```diff
@@ -15,30 +17,25 @@ class CallbacksController < VANotify::ApplicationController
 
     service_tag 'va-notify'
 
-    skip_before_action :verify_authenticity_token, only: [:create]
-    skip_before_action :authenticate, only: [:create]
+    skip_before_action :verify_authenticity_token, only: [:update]
+    skip_before_action :authenticate, only: [:update]
 
-    before_action :set_notification, only: [:create]
-    before_action :authenticate_callback!, only: [:create]
+    before_action :set_notification, only: [:update]
+    before_action :authenticate_callback!, only: [:update]
 
-    def create
+    def update
       notification_id = params[:id]
       if @notification
```

Comment:
Instead of attempting to look up the notification and then falling back to the async job when the lookup fails I think we should make everything async (behind a feature flag).

Reply:
I've placed it behind a (new) feature flag, though I considered placing it behind an existing feature flag to coordinate with request_level_callbacks. 

I also moved my interpretation of "everything" into the job